apps/em/em-anno/demo.yaml

```diff
@@ -7,6 +7,6 @@ spec:
     java:
       disableIngressClassAnnotation: true
       ingressClass: traefik-private
-      image: hmctspublic.azurecr.io/em/anno:prod-0cc0d7e-20221216091410 #{"$imagepolicy": "flux-system:em-anno"}
+      image: hmctspublic.azurecr.io/em/anno:pr-1113-495182f-20221216110225 #{"$imagepolicy": "flux-system:em-anno"}
```

Comment:
#{"$imagepolicy": "flux-system:em-anno"}.  should be #{"$imagepolicy": "flux-system:demo-em-anno"}

Reply:
updated

Reply:
updated